### PR TITLE
[ci] Add age to CI deploy/build statuses

### PR DESCRIPTION
**Pull request by tpoterba**
stacked on #6344

### CODE REVIEW

**Comment by danking** [comment]
There's something weird about this diff

**Comment by tpoterba** [comment]
oh, oops. I must have rebased master onto the other branch. I'll fix it up when the parent goes in.

File: ci/ci/ci.py [hunk 1]
```diff
@@ -68,7 +69,8 @@
             'deploy_batch_id': wb.deploy_batch.id if wb.deploy_batch and hasattr(wb.deploy_batch, 'id') else None,
             'deploy_state': wb.deploy_state,
             'repo': wb.branch.repo.short_str(),
-            'prs': pr_configs
+            'prs': pr_configs,
+            'status_Age': wb.pretty_status_age(),
```
Comment (danking) [inline ci/ci/ci.py]: this capital A looks like a bug

Comment (tpoterba) [reply ci/ci/ci.py]: yes!

**Pull request merged**